badge.html

```diff
@@ -0,0 +1,113 @@
+
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Badge</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header class="nav-header">
+        <div class="nav-header-div">
+            <div class="nav-header-div-lt">
+                <a href="index.html">Home</a>
+                <a href="#">Products</a>
+                <a href="#">Docs</a>
+                <a href="#">Pricing</a>
+                <a href="#">About us</a>
+                <a href="#">Blogs</a>
+            </div>
+            <div class="nav-header-div-rt">
+                <button>Search</button>
+                <button>Github</button>
+                <button>Dark mode</button>
+            </div>
+        </div>
+    </header>
+
+    <div class="view">
+        <aside>
+            <div class="nav-pill">
+                <a href="docs.html">Avatar</a>
+                <a class="active" href="badge.html">Badge</a>
+                <a href="#">Tabs</a>
+                <a href="#">Card</a>
+                <a href="#">Dialog</a>
+            </div>
+        </aside>
+
+        <main class="main-view">
+            <div class="main-view-container">
+                <h1>Badge</h1>
+                <p>Badge generates a small badge to the top-right of its child(ren).</p>
+                <h2>Basic badge</h2>
+                <p>Examples of badges containing text, using primary and secondary colors. The badge is applied to its children.</p>
+                <div class="component-container">
+                    <div class="badge-container">
+                        <p class="badge-left" style="padding: 0; margin: 0; font-weight: bold;">Make</p>
+                        <span>4</span>
+                    </div>
+                
+                    <div class="badge-container">
+                        <p class="badge-left" style="padding: 0; margin: 0; font-weight: bold;">Your</p>
+                        <span>5</span>
+                    </div>
+
+                    <div class="badge-container">
+                        <p class="badge-left" style="padding: 0; margin: 0; font-weight: bold;">Way</p>
```

Comment:
Can avoid inline styling for all p tags containing 'badge-left' class and instead can declare all the same style properties here  in a  separate class and use the class here

Reply:
Declared a separate class p-bold for this inline styling.